gateway-contracts/tasks/deployment/contracts.ts

```diff
@@ -199,5 +222,53 @@ task("task:deployAllGatewayContracts").setAction(async function (_, hre) {
   console.log("Deploy Decryption contract:");
   await hre.run("task:deployDecryption");
 
+  console.log("Deploy ProtocolPayment contract:");
+  await hre.run("task:deployProtocolPayment");
+
   console.log("Contract deployment done!");
 });
+
+// Deploy all the contracts
+task("task:deployAllGatewayContracts").setAction(async function (_, hre) {
+  // Deploy all the setup contracts
+  await hre.run("task:deploySetupContracts");
+
+  // Deploy all the regular contracts
+  await hre.run("task:deployRegularContracts");
```

Comment:
nit: not sure about the "regular" wording here. Another idea could be `deployBaseContracts` for all initially required contracts and `deployGatewayContracts` for these "regular" ones.